src/components/MovieDetails/MovieDetails.js

```diff
@@ -20,7 +20,7 @@ const MovieDetails = () => {
 
   return (
     <div className="movie-details">
-      <div className="movie-details__nav">
+      <div className="details-header">
```

Comment:
`details` are very generic, but you're here in the specific component already.

```suggestion
      <div className="movie-header">
```